crates/net/network/src/session/mod.rs

```diff
@@ -427,6 +426,14 @@ impl SessionManager {
     }
 }
 
+/// A message to update the status.
+#[derive(Debug, Clone)]
+pub(crate) struct StatusUpdate {
+    pub(crate) height: u64,
+    pub(crate) hash: H256,
+    pub(crate) total_difficulty: U256,
+}
+
```

Comment:
I don't think this fits here anymore.

I think named fields in the message variant would be more fitting here